### PR TITLE
Mobile login hotfix

### PR DESCRIPTION
**Pull request by dark-r00t**
I changed the login screen to reflect the values it actually required. Additionally, I used .trim() to remove white spaces from the email prior to sending the input off.

### CODE REVIEW

**Comment by dark-r00t** [comment]
From Railway:

#13 11.50 ./src/pages/api/user/register.ts:9:26
#13 11.50 Type error: Module '"../../../utils/validation"' has no exported member 'validateString'.
#13 11.50
#13 11.50    7 | import { prisma } from "../../../server/db/client";
#13 11.50    8 | import { generateJWTForEmailVerification } from "../../../utils/jwt";
#13 11.50 >  9 | import { validateMethod, validateString } from "../../../utils/validation";
#13 11.50      |                          ^
#13 11.50   10 | 
#13 11.50   11 | interface Request extends NextApiRequest {


I don't think this is an issue with my code. I never touched this file...

File: mobile/package.json [hunk 1]
```diff
@@ -28,10 +28,11 @@
     "jest": "^26.6.3",
     "jest-expo": "^46.0.1",
     "nativewind": "^2.0.11",
+    "ngrok": "^4.1.0",
     "react": "18.0.0",
     "react-dom": "18.0.0",
     "react-native": "0.69.6",
-    "react-native-gesture-handler": "^2.8.0",
+    "react-native-gesture-handler": "~2.5.0",
```
Comment (kitorer) [inline mobile/package.json:35]: wait is this a downgrade?

File: mobile/src/pages/Login.tsx [hunk 2]
```diff
@@ -62,7 +62,7 @@ export default function Login({ navigation }) {
             disabled={isPending || invalidCount >= 5}
             onPress={() => {
               setIsPending(true);
-              login(username, password)
+              login(email.trim(), password)
```
Comment (kitorer) [inline mobile/src/pages/Login.tsx:65]: wait Ive got the feeling that if the api let you submit here or in password with an extra space shouldnt that trim be in web/scr/api?

**Review by kitorer — APPROVED** [review]
actual code is good. _just some suggestions_

**Comment by dark-r00t** [comment]
Let the server handle trim.

**Pull request closed without merging**